google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/AbstractMockServerTest.java

```diff
@@ -114,6 +114,8 @@ public abstract class AbstractMockServerTest {
       Statement.of("INSERT INTO TEST (ID, NAME) VALUES (1, 'test aborted')");
   public static final Statement INSERT_RETURNING_STATEMENT =
       Statement.of("INSERT INTO TEST (ID, NAME) VALUES (1, 'test aborted') THEN RETURN *");
+  public static final Statement PG_INSERT_RETURNING_STATEMENT =
```

Comment:
Unrelated to delayed transactions?

Reply:
I added this so we could include tests that execute DML with a RETURNING clause in both dialects. The reason that it is relevant is that these statements are executed using the `executeQuery` method, and we should ensure that these statements trigger the start of the underlying transaction, even though they are executed through the `executeQuery` method.

So this statement is referenced in the `DelayTransactionStartUntilFirstWriteMockServerTest`